python/tvm/relay/frontend/pytorch.py

```diff
@@ -2241,7 +2247,7 @@ def convert_operators(operators, outputs, ret_names, convert_map, prelude):
                 out_names = _get_output_names(op_node)
                 outputs.update(zip(out_names, relay_out))
             else:
-                outputs[node_name] = relay_out
+                outputs[_get_output_name(op_node)] = relay_out
```

Comment:
I think it is clearer to directly assert `op_node.outputsSize() == 1` here, instead of calling `_get_output_name` again.

Reply:
Thanks, done!